src/xcb_util.cpp

```diff
@@ -267,6 +267,12 @@ ewmh_change_desktop (xcb_connection_t *c, xcb_window_t root, uint destkop_id)
   // of this spec, in which case the timestamp field should be ignored.
   send_xcb_message (c, root, "_NET_CURRENT_DESKTOP", { destkop_id });
 }
+/**
+ * Get information about 
+ * all windows, including
+ * ids, related desktops,
+ * geometries and icons 
```

Comment:
Why 20 character limit?

And you better leave an empty line before the documentation